composer.json

```diff
@@ -33,7 +33,7 @@
         }
     ],
     "require": {
-        "php": "~8.2.0 || ~8.3.0",
+        "php": ">=8.2",
```

Comment:
Please do not put an open ended constraint here, the limited support to new PHP versions is intentional